code/datums/elements/deployable_item.dm

```diff
@@ -117,8 +117,8 @@
 
 //Handles the conversion of Machine into Item. 'source' should be the Machine. User is the one undeploying. It can be undeployed without a user, if so, the var 'location' is required. If 'source' is not /obj/machinery/deployable then 'undeploying' should be the item to be undeployed from the machine.
 /datum/element/deployable_item/proc/finish_undeploy(datum/source, mob/user)
-	var/obj/machinery/deployable/deployed_machine = source //The machinethat is undeploying should be the the one sending the Signal
-	var/obj/item/attached_item  = deployed_machine.internal_item //Item the machine is undeploying
+	var/obj/deployed_machine = source //The machinethat is undeploying should be the the one sending the Signal
+	var/obj/item/attached_item  = deployed_machine:internal_item //Item the machine is undeploying
```

Comment:
absolutely not

Reply:
Ive thought about it and putting it on obj is probably teh best idea but you should probably ask brave and spooky for second opinions